spec/form_models/jobseekers/job_application/personal_details_form_spec.rb

```diff
@@ -109,6 +109,5 @@
   it { is_expected.to allow_value("01234 12345678").for(:phone_number) }
   it { is_expected.not_to allow_value("01234 123456789").for(:phone_number) }
 
-  it { is_expected.to validate_inclusion_of(:right_to_work_in_uk).in_array(%w[yes no]) }
```

Comment:
Aren't we missing the validation test over `has_right_to_work_in_uk` ?

Reply:
no, you can't true/false validations like this - you get warnings about the test being pointless on a boolean

Reply:
Oh, thanks for the context!